test/passes/O3_low-memory-unused_metrics.txt

```diff
@@ -444,11 +434,24 @@ total
                       (local.get $3)
                      )
                     )
+                    (local.set $4
+                     (i32.load offset=36
+                      (local.get $3)
+                     )
+                    )
```

Comment:
It's hard to tell from these diffs whether this movement is correct. I assume the fuzzer would find any problems very quickly, though.